bundle/manifests/poison-pill.medik8s.io_poisonpillremediations.yaml

```
@@ -37,6 +37,18 @@ spec:
             type: object
           spec:
             description: PoisonPillRemediationSpec defines the desired state of PoisonPillRemediation
+            properties:
+              remediationStrategy:
+                default: NodeDeletion
+                description: RemediationStrategy is the remediation method for unhealthy
+                  nodes could be either "NodeDeletion" or "ResourcesDeletion" the
+                  first will delete the node to signal to the cluster that the node
+                  was fenced the latter will iterate over all pos and volumeattachments
+                  related to the unhealthy node and delete them
+                enum:
+                - NodeDeletion
+                - ResourcesDeletion
```

Singular for consistency

```suggestion
                - ResourceDeletion
```

---

I was wondering about that.
we delete a single node, but multiple resources (pods and volume).
would you still think that the singular form is better?

---

fixed